content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -219,7 +219,7 @@ option. Your cluster requirements may need a different configuration.
      ```
 
    {{< note >}}
-   The `kubeadm-certs` Secret and decryption key expire after two hours.
+   The certificate key is a hex encoded string that is an AES key of size 32 bytes.
```

Comment:
please add new note instead of deleting the old one.

/sig cluster-lifecycle

Reply:
Hey @neolit123 ! Please have a just pushed the latest changes . Does this make sense ?